ambry-api/src/main/java/com.github.ambry/rest/RestResponseChannel.java

```diff
@@ -97,11 +97,24 @@ public void setStatus(ResponseStatus status)
       throws RestServiceException;
 
   /**
+   * Gets the current {@link ResponseStatus}.
+   * @return the response status.
+   */
+  public ResponseStatus getStatus();
```

Comment:
Yeah that is true for all interfaces. But we have followed the practice of including the keyword `public` in every interface. This is to maintain consistency.